platform/core/src/classes/HotkeysManager.js

```diff
@@ -1,3 +1,4 @@
+import cloneDeep from 'lodash.clonedeep';
```

Comment:
Because we're cloning something simple, and we know the shape, I wonder if we can do it without pulling in this dep and in a more performant way.

Not blocking. Just food for thought.